setup.cfg

```diff
@@ -6,6 +6,12 @@ rednose=1
 # Uncomment the following line to open pdb when a test fails
 #pdb=1
 
-[pep8]
-ignore=E501,E402
+[pycodestyle]
+# error codes: https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
+# E402: module level import not at top of file
+#   We catch this with pylint, don't need 2 reports of the same issue
+# E501: line too long
+# E722: do not use bare except, specify exception instead
+#   We catch this with pylint, don't need 2 reports of the same issue
+ignore=E501,E402,E722
```

Comment:
So pycodestyle is a drop-in replacement for PEP-8? Is it our new best practice?

Reply:
The package was renamed at Guido's request: https://github.com/pycqa/pycodestyle

> This package used to be called pep8 but was renamed to pycodestyle to reduce confusion. Further discussion can be found in the issue where Guido requested this change, or in the lightning talk at PyCon 2016 by @IanLee1521: slides video.